composer.json

```diff
@@ -12,7 +12,7 @@
     ],
 
     "require": {
-      "php": "^7.1"
+      "php": "^8.0"
```

Comment:
Hello @s-gonchar 
First at all, thanks a lot for your contribution!!!

It seems that is not enought changing version of php: Tests fail

Rnning with php-8.0.9-nts-Win32-vs16-x64

`
$ vendor/bin/phpunit
PHPUnit 8.4-gf12f04116 by Sebastian Bergmann and contributors.

EEEEEEEEEEEEEEEEEEEEEEEE                                          24 / 24 (100%)

Time: 7.22 seconds, Memory: 4.00 MB

There were 24 errors:

1) StateMachineTest::test_can_make_StateMachine_class
PHPUnit\Framework\Exception: El nombre de archivo, el nombre de directorio o la sintaxis de la etiqueta del volumen no son correctos.
[snip]
`
Trans:
PHPUnit\Framework\Exception: The file name, directory name, or volume label syntax is not correct.

With php7.2.8 goes fine:

`
$ vendor/bin/phpunit
PHPUnit 8.4-gf12f04116 by Sebastian Bergmann and contributors.

........................                                          24 / 24 (100%)
My current state is 1 on A and 2 will be the next state
My current state is 2 on B and 3 will be the next state
My current state is 3 on C and 1 will be the next state
My current state is 1 on A and 2 will be the next state
My current state is 2 on B and 3 will be the next state
My current state is 3 on C and 1 will be the next state

Time: 37.53 seconds, Memory: 2.00 MB

OK (24 tests, 36 assertions)
`

Samething occures around PHPUnit\Framework\Exception when I am using \Exception just once in StateMachineTest on line 17
I keep investigating...